### PR TITLE
reco.py: change validate api key endpoint

### PR DESCRIPTION
**Pull request by GalNakash-RecoLabs**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Review by maimorag — COMMENTED** [review]
Hey @GalNakash-RecoLabs,
Thank you for your contribution!
Good work :)

The docker image tag is not the latest numeric tag, please update it.
To update the docker image run:
`demisto-sdk format -ud -i Packs/Reco/Integrations/Reco/Reco.yml
`

Please feel free to reach out to me with any questions - I'm available here or on slack :)
Thanks again :)

File: Packs/Reco/ReleaseNotes/1_2_1.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### Reco
+
+- Change validate api key to use the new Reco API key validation endpoint
```
Comment (maimorag) [inline Packs/Reco/ReleaseNotes/1_2_1.md:6]:
```suggestion
- Updated the validate api key to use the new Reco API key validation endpoint.

```

File: Packs/Reco/ReleaseNotes/1_2_1.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+
+#### Integrations
+
+##### Reco
+
+- Updated the validate api key to use the new Reco API key validation endpoint.
+
```
Comment (maimorag) [inline Packs/Reco/ReleaseNotes/1_2_1.md:7]:
```suggestion
- Updated the validate api key to use the new Reco API key validation endpoint.
- Updated the Docker image to: *demisto/python3:3.10.14.91134*.

```

**Comment by maimorag** [comment]
Hi @GalNakash-RecoLabs ,
the code looks good!

We're ready for a demo. Please check [this page](https://xsoar.pan.dev/docs/contributing/demo-prep), and let me know when you're available for one over DFIR.  
You can also send me a recording of a demo.

**Comment by GalNakash-RecoLabs** [comment]
It's just test so I will share a video soon

Sent via Superhuman ***@***.***>

File: Packs/Reco/ReleaseNotes/1_2_1.md (hunk 2 above)
Comment (maimorag) [reply Packs/Reco/ReleaseNotes/1_2_1.md:7]: > 

Please notice this comment as well, and add the updated docker image note.

File: Packs/Reco/Integrations/Reco/Reco_test.py [hunk 3]
```diff
@@ -420,7 +420,7 @@ def get_mock_assets() -> list[dict[str, Any]]:
 def test_test_module_success(requests_mock, reco_client: RecoClient) -> None:
     mock_response = {"dataSources": {"tablesMetadata": [{"name": "table1"}]}}
     requests_mock.get(
-        f"{DUMMY_RECO_API_DNS_NAME}/data-sources", json=mock_response
+        f"{DUMMY_RECO_API_DNS_NAME}/policy-subsystem/alert-inbox?limit=1", json=mock_response
```
Comment (maimorag) [inline Packs/Reco/Integrations/Reco/Reco_test.py:423]: Hey Please notice you have an assertion error in the unit test test_test_module_success
` AssertionError: assert 'Invalid token' == 'ok'`

**Review by maimorag — APPROVED** [review]

**Pull request merged**